src/main/java/com/pileproject/drivecommand/machine/device/input/simulation/SimulatedColorSensor.java

```diff
@@ -0,0 +1,38 @@
+package com.pileproject.drivecommand.machine.device.input.simulation;
```

Comment:
I feel these objects are just for mocking. What do you think about moving to com.pileproject.drivecommand.**mock**.machine.device.input ?